### PR TITLE
Added Install button on toolbar when selecting INF files

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Related #7798

**Details of Changes**
Add details of changes here.
- Added install button on toolbar when selecting INF files.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -285,6 +285,17 @@
...
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="InstallInfButton"
+                    x:Load="{x:Bind ViewModel.IsInfFile, Mode=OneWay, FallbackValue=False}"
+                    Command="{x:Bind ViewModel.InstallInfCommand, Mode=OneWay}"
+                    Label="{helpers:ResourceString Name=Install}"
+                    LabelPosition="Default"
+                    ToolTipService.ToolTip="{helpers:ResourceString Name=Install}">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xE91B;" />
```
Comment (yaichenbaum) [inline src/Files/UserControls/InnerNavigationToolbar.xaml:296]:
```suggestion
                        <FontIcon Glyph="&#xE9F5;" />
```
image

**Comment by puppetsw** [comment]
Might be best to use InfDefaultInstall.exe - it's what explorer uses to install the drivers from the right click context menu.

File: src/Files/Interacts/BaseLayoutCommandImplementationModel.cs [hunk 2]
```diff
@@ -787,6 +787,25 @@ public async void DecompressArchiveToChildFolder()
             }
         }
 
+        public async void InstallInfDriver()
+        {
+            foreach (ListedItem selectedItem in SlimContentPage.SelectedItems)
+            {
+                var connection = await AppServiceConnectionHelper.Instance;
```
Comment (d2dyno1) [inline src/Files/Interacts/BaseLayoutCommandImplementationModel.cs:794]: You could move this up so it isn't called everytime as well as the if statement

File: src/Files/Interacts/BaseLayoutCommandImplementationModel.cs [hunk 3]
```diff
@@ -787,6 +787,25 @@ public async void DecompressArchiveToChildFolder()
             }
         }
 
+        public async void InstallInfDriver()
```
Comment (d2dyno1) [inline src/Files/Interacts/BaseLayoutCommandImplementationModel.cs:790]: Change to `public async Task`

File: src/Files/Interacts/BaseLayoutCommandsViewModel.cs [hunk 4]
```diff
@@ -72,6 +72,7 @@ private void InitializeCommands()
             DecompressArchiveCommand = new RelayCommand(CommandsModel.DecompressArchive);
             DecompressArchiveHereCommand = new RelayCommand(CommandsModel.DecompressArchiveHere);
             DecompressArchiveToChildFolderCommand = new RelayCommand(CommandsModel.DecompressArchiveToChildFolder);
+            InstallInfDriver = new RelayCommand(CommandsModel.InstallInfDriver);
```
Comment (d2dyno1) [inline src/Files/Interacts/BaseLayoutCommandsViewModel.cs:75]: `new AsyncRelayCommand()`

File: src/Files/UserControls/InnerNavigationToolbar.xaml [hunk 5]
```diff
@@ -288,6 +288,17 @@
                         <FontIcon Glyph="&#xE91B;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
```
Comment (d2dyno1) [inline src/Files/UserControls/InnerNavigationToolbar.xaml:291]: Add AutomationProperties.Name

**Review by d2dyno1 — CHANGES_REQUESTED** [review]

File: src/Files/Interacts/BaseLayoutCommandImplementationModel.cs (hunk 2 above)
Comment (puppetsw) [reply src/Files/Interacts/BaseLayoutCommandImplementationModel.cs:794]: Ah yes, we should probably update the other similar methods like this as well.

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**